pkg/instance/handler.go

```diff
@@ -209,16 +210,16 @@ func (h Handler) consumeParameters(user *model.User, sourceInstanceId uint, inst
 	}
 
 	if preset && !sourceInstance.Preset {
-		return apperror.NewUnauthorized(fmt.Sprintf("instance (id: %d) isn't a preset", sourceInstance.ID))
+		return errdef.NewUnauthorized(fmt.Sprintf("instance (id: %d) isn't a preset", sourceInstance.ID))
```

Comment:
I don't think you need to call `fmt.Sprintf` as it will delegate to `fmt.Errorf` anyway